src/go/rpk/pkg/cli/topic/config.go

```diff
@@ -72,37 +72,6 @@ valid, but does not apply it.
 			subtractKVs, err := parseKVs(subtracts)
 			out.MaybeDie(err, "unable to parse --subtract: %v", err)
 
-			// Redpanda fails to set both remote.read and write when passed
```

Comment:
I will defer to you or others (@r-vasquez?) on this. It would be nice to deprecate this code in the same release as the fix, but I understand the need for backwards compatibility here.

Reply:
We will keep this code for now and remove it in a future major release.
Added a `TODO` here instead.